### PR TITLE
feat(RSS3): mf-3770 add deposit, withdraw, deposit and staking

### PR DESCRIPTION
**Pull request by UncleBill**
https://mask.atlassian.net/browse/MF-3770

### CODE REVIEW

File: packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/TokenOperationCard.tsx [hunk 1]
```diff
@@ -96,6 +108,9 @@ export const TokenOperationCard: FC<TokenFeedCardProps> = ({ feed, ...rest }) =>
                             to,
                             value: formatValue(metadata),
                             symbol: metadata!.symbol,
+                            /* eslint-disable-next-line  @typescript-eslint/ban-ts-comment */
+                            // @ts-ignore
+                            exchange: action.platform!,
```
Comment (UncleBill) [inline packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/TokenOperationCard.tsx:113]: `i18n-code-generator` should extract typings from nested translation

```diff
--- a/packages/plugins/RSS3/src/locales/en-US.json
+++ b/packages/plugins/RSS3/src/locales/en-US.json
@@ -3,10 +3,14 @@
     "token_operation$send": "<from>{{from}}</from> sent to <to>{{to}}</to>",
     "token_operation$claim": "<to>{{to}}</to> claim from <from>{{from}}</from>",
     "token_operation$burn": "<from>{{from}}</from> burned <bold>{{value}} {{symbol}}</bold>",
+    "token_operation$deposit": "<from>{{from}}</from> deposited on <bold>{{exchange}}</bold>",
+    "token_operation$withdraw": "<from>{{from}}</from> withdrew on <bold>{{exchange}}</bold>",
     "token_operation_verbose$mint": "$t(token_operation$mint)",
     "token_operation_verbose$send": "<from>{{from}}</from> sent to <to>{{to}}</to> for <bold>{{value}} {{symbol}}</bold>",
     "token_operation_verbose$claim": "<to>{{to}}</to> claim from <from>{{from}}</from> for <bold>{{value}} {{symbol}}</bold>",
     "token_operation_verbose$burn": "$t(token_operation$burn)",
+    "token_operation_verbose$deposit": "$t(token_operation$deposit)",
+    "token_operation_verbose$withdraw": "$t(token_operation$withdraw)",
     "token_approval$approve": "<bold>{{user}}</bold> approved <bold>{{- amount}} {{symbol}}</bold> to <bold>{{contract}}</bold>",
     "token_approval$revoke": "<bold>{{user}}</bold> revoked the approval of <bold>{{- amount}} {{symbol}}</bold> to <bold>{{contract}}</bold>",
     "token_approval_verbose$approve": "<bold>{{user}}</bold> approved <bold>{{- amount}} {{symbol}}</bold>",
```

Comment (UncleBill) [reply packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/TokenOperationCard.tsx:113]: @Jack-Works ⬆️

Comment (UncleBill) [reply packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/TokenOperationCard.tsx:113]: image

**Review by guanbinrui — APPROVED** [review]

File: packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/TokenOperationCard.tsx (hunk 1 above)
Comment (Jack-Works) [reply packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/TokenOperationCard.tsx:113]: ack, I'll open an issue and handle that later

**Pull request merged**